CityOfDetroit/bloom
# 992/public primary unit group

## Issue

- Addresses #992

## Description

This adds the primary unit group summary to the listing detail page.

![Screen Shot 2022-03-07 at 10 47 18 PM](https://user-images.githubusercontent.com/4401157/157174773-c21e30a0-9c5b-4194-9a31-0d5333e936a6.png)

A few things to note while reviewing this:
1. for the sake of time and because there are other unit group summaries to still be built I did not abstract helper functions where we will likely want to in both the front end and backend
2. I did not include the info icon next to AMI, because that requires an update to the StandardTable element as well as the info icon and I think that should be done separately

This also adds household maximum income.

![Screen Shot 2022-03-08 at 2 23 09 PM](https://user-images.githubusercontent.com/4401157/157327434-8fa10007-7da2-4e4d-9eed-262f30ba1e6a.png)


## Type of change


## How Can This Be Tested/Reviewed?

Pull this down and run `yarn db:reseed:detroit`. On the public side, find the listing titled `MARTIN LUTHER KING II`. Go to the detail, and you should see the table.

## Checklist:
